webui-user.sh

```diff
@@ -10,7 +10,7 @@
 #clone_dir="stable-diffusion-webui"
 
 # Commandline arguments for webui.py, for example: export COMMANDLINE_ARGS="--medvram --opt-split-attention"
-#export COMMANDLINE_ARGS=""
+export COMMANDLINE_ARGS="--listen"
```

Comment:
I believe there's a rule we shouldn't change user config files because it may override existing user configs

and even we can change that I don't think adding `--listen` to the default profile config is a good idea

Reply:
+1 for @w-e-w 

my suggestion is to modify your last line of Dockerfile to 
```
CMD [ "./webui.sh", "--skip-prepare-environment" , "--listen"]
```